capstone-app/src/app/modules/history/wiki-media/wiki-media.component.html

```diff
@@ -4,9 +4,12 @@
       <mat-card-title class="history-title">
         {{title}}
       </mat-card-title>
-      <div class="text">
-      {{body}}
+      <div class="text" *ngFor="let paragraph of body">
+        <br/>
```

Comment:
Recommend don't use line break element. Instead, increase top margin on the paragraph element below.